docs/components/icon.md

```diff
@@ -28,7 +28,7 @@ The name of the icon. This required prop will only recognize icons from the foll
             :class="{ hasBg: isToggled.value }"
           >
             <KIcon
-            size="24"
+            size="24px"
```

Comment:
Because we're now using `KUI_ICON_SIZE_50 = "24px";` which are in px  ref: https://github.com/Kong/design-tokens/blob/main/TOKENS.md?plain=1#L1291
also following what Max's example https://github.com/Kong/kongponents/pull/1558/files#r1270942493